cypress-tests/cypress/support/commands.js

```diff
@@ -847,9 +850,18 @@ Cypress.Commands.add(
             expect(res_data.body[key]).to.equal(response.body[key]);
           }
         } else if (response.body.capture_method === "manual") {
-          expect(response.body).to.have.property("mandate_id");
+          if (requestBody.mandate_data === null) {
+            expect(response.body).to.have.property("payment_method_id");
+          } else {
+            expect(response.body).to.have.property("mandate_id");
+          }
```

Comment:
we can remove these as it is handled above

Reply:
done